libs/core/execution_base/include/hpx/execution_base/traits/coroutine_traits.hpp

```diff
@@ -0,0 +1,220 @@
+//  Copyright (c) 2022 Shreyas Atre
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/config/coroutines_support.hpp>
+#include <hpx/config/forward.hpp>
```

Comment:
This change would simply things:
```suggestion
#include <hpx/config.hpp>
```